docs/extensiondev.rst

```diff
@@ -399,20 +396,14 @@ extension to be approved you have to follow these guidelines:
 Extension Import Transition
 ---------------------------
 
-For a while we recommended using namespace packages for Flask extensions.
-This turned out to be problematic in practice because many different
-competing namespace package systems exist and pip would automatically
-switch between different systems and this caused a lot of problems for
-users.
-
-Instead we now recommend naming packages ``flask_foo`` instead of the now
-deprecated ``flaskext.foo``.  Flask 0.8 introduces a redirect import
-system that lets uses import from ``flask.ext.foo`` and it will try
-``flask_foo`` first and if that fails ``flaskext.foo``.
+Flask 0.8 introduced a redirect import system that lets users import from
+``flask.ext.foo`` before trying ``flask_foo`` and if that fails
+``flaskext.foo``. As of Flask 1.0 this import redirection system will be
```

Comment:
Since these are the 1.0 docs, we can write in the past tense: "The import redirection system has been removed"